main.js

```diff
@@ -1,3 +1,6 @@
+var driving;
```

Comment:
Make state internal to your car. Otherwise, your program cannot support multiple, independent cars.